Nuget/Ploeh.AutoFixture.nuspec

```diff
@@ -6,6 +6,9 @@
         <title>AutoFixture</title>
         <authors>Mark Seemann</authors>
         <owners>Mark Seemann</owners>
+        <dependencies>
+            <dependency id="Albedo" version="[1,2)" />

```

Comment:
I think it ought to reference the lowest possible version that we know works. If we're using 1.0.1, we should state that as a requirement.

If we wish to be able to state that 1.0.0 is the minimum version supported, we should downgrade our use of Albedo so that we are, in fact, testing against that version.